site/src/data/archives.data.ts

```diff
@@ -2,6 +2,16 @@ import { Archive, ArchiveCategory } from '~/types/library';
 import { contributorsData } from '~/data/contributors.data';
 
 export const archives: Array<Archive> = [
+  {
+    id: '74a6440375df7b5c',
+    title: 'Nervos Network Developers Resources Hub',
```

Comment:
```suggestion
    title: 'Nervos Network Developer Resources Hub',
```

Reply:
Isn't this supposed to be consistent with the original article title?